src/credentials/attestation_verifier/DeviceAttestationVerifier.h

```diff
@@ -146,6 +146,25 @@ class AttestationTrustStore
      *
      */
     virtual CHIP_ERROR GetProductAttestationAuthorityCert(const ByteSpan & skid, MutableByteSpan & outPaaDerBuffer) const = 0;
+
+    /**
+     * @brief Look-up a CD cert by SKID
+     *
+     * The implementations of this interface must have access to a set of CDs to trust.
+     *
+     * Interface is synchronous, and therefore this should not be used unless to expose a CD
+     * store that is both fully local and quick to access.
+     *
+     * @param[in] skid Buffer containing the subject key identifier (SKID) of the CD to look-up
+     * @param[in,out] outCDDerBuffer Buffer to receive the contents of the CDD cert, if found.
+     *                               Size will be updated to match actual size.
+     *
+     * @returns CHIP_NO_ERROR on success, CHIP_INVALID_ARGUMENT if `skid` or `outCDDerBuffer` arguments
+     *          are not usable, CHIP_BUFFER_TOO_SMALL if certificate doesn't fit in `outCDDerBuffer`
+     *          span, CHIP_ERROR_CA_CERT_NOT_FOUND if no PAA found that matches `skid.
+     *
+     */
+    virtual CHIP_ERROR GetCertificationDeclarationCert(const ByteSpan & skid, MutableByteSpan & outCDDerBuffer) const = 0;
```

Comment:
Since most implementations seem to do not-implemented, how about not making this pure virtual and instead do the `return CHIP_ERROR_NOT_IMPLEMENTED` here?